README.md

```diff
@@ -220,67 +220,31 @@ Used on iOS. If YES, the network is WEP Wi-Fi; otherwise it is a WPA or WPA2 per
 ## Only Android
 The following methods work only on Android
 
-### `loadWifiList(successCallback: function, errorCallback: function)`
-
-Method to get a list of nearby WiFI networks.
-
-#### successCallback( wifiList:  string )
-
-Type: `function`
-
-Function to be called if the attempt is successful. It contains a stringified JSONArray of wifiObjects as parameter, each object containing:
-
+<details>
+<summary>loadWifiList(): Promise<Array<WifiEntry>></summary>
+Returns a list of nearby WiFI networks.
 * `SSID`: The network name.
 * `BSSID`: The WiFi BSSID.
 * `capabilities`: Describes the authentication, key management, and encryption schemes supported by the access point.
 * `frequency`: The primary 20 MHz frequency (in MHz) of the channel over which the client is communicating with the access point.
 * `level`: The detected signal level in dBm, also known as the RSSI.
 * `timestamp`: timestamp in microseconds (since boot) when this result was last seen.
+</details>
```

Comment:
This XML-like display does not look right on GitHub web, could you check it out @eliaslecomte?

Reply:
I've updated the docs.